tests/test_spawn.py

```diff
@@ -35,3 +36,15 @@ def test_spawn_unicode_sh(self):
         
         with self.assertRaises(EOFError):
             p.read()
+
+    def test_quick_spawn(self):
+        """Spawn a very short-lived process."""
+        # so far only reproducable on Solaris 11, spawning a process
+        # that exits very quickly raised an exception at 'inst.setwinsize',
+        # because the pty filedes was quickly lost after exec().
```

Comment:
Is filedes a common abbreviation? It took me a moment to work it out - I'm more used to seeing `fd`. This doesn't matter, though. :-)